creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -126,7 +126,31 @@ const retrieveClockValueForUserFromReplica = async (replica, wallet) => {
   return clockValue
 }
 
+const processJob = async (
```

Comment:
with functions like this that are very important structurally, i'm starting to feel its not great to hide them inside a large utils file. utils should be more for simple pure functions unrelated to business logic like "handlePromise" or "convertObjToArr" etc

this one is crucial business logic, prob should have it as own file

Reply:
would you prefer this to go into something like `src/services/stateMachineManager/processJob.js` or in something to be shared with other jobs in the future -- like a `src/utils/bull.js` file? I think you mentioned something like the latter before